docs/docs/how-tos/use-gpus.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: use-gpus
+title: Using GPUs on Nebari
+description: Provide a complete overview of using GPUs on Nebari, including server setup, environment setup, and validation.
+---
+
+# Using GPUs on Nebari
+## Introduction
+Provide a complete overview of using GPUs on Nebari, including server setup, environment setup, and validation.
+
+## 1. Starting a GPU server
+
+Follow Steps 1 to 3 in the [Authenticate and launch JupyterLab][login-with-keycloak] tutorial. The UI will show a list of profiles (instances, servers, machines).
+
+![Nebari select profile](/img/how-tos/nebari_select_profile.png)
+Your administrator pre-configures these options, as described in [Profile Configuration documentation][profile-configuration].
+
+Select an appropriate profile and click "Start".
+
+### Understanding GPU setup on the server.
+The following steps describe how to get CUDA-related information from the server.
+1. Once your server starts, it will redirect you to a JupyterLab home page.
+2. Click on the **"Terminal"** icon.
+3. Run the command `nvidia-smi`. The top right corner of the command's output should have the highest supported driver.
+    ![nvidia-smi-output](/img/how-tos/nvidia-smi-output.png)
+
+    If you get `nvidia-smi: command not found`. You are most likely on a non-GPU server. Shutdown your server, and start up a GPU-enabled server.
+
+## 2. Creating environments
+
+By default, `conda-store` will build CPU-compatible packages. To build GPU-compatible packages, we the following.
+### Build a GPU-compatible environments
+By default, conda-store will build CPU-compatible packages. To build GPU-compatible packages, we have a couple of options:
+1. **Creating config using PyTorch documentation**:
```

Comment:
We need to swap these two sections (1 & 2). We want approach 2 to be the way folks do it, but just make them aware that approach 1 also exists.